daras_ai_v2/facebook_bots.py

```
@@ -418,19 +421,23 @@ def send_wa_msgs_raw(
-        f"https://graph.facebook.com/v16.0/{bot_number}/messages",
-        headers=get_wa_auth_header(access_token),
-        json={
-            "messaging_product": "whatsapp",
-            "status": "read",
-            "message_id": message_id,
-        },
-    )
-    print("wa_mark_read:", r.status_code, r.json())
+    for message_id in message_ids:
+        # send read receipt
+        r = requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)